cpp/epidemiology/utils/parameter_distributions.h

```diff
@@ -288,6 +288,12 @@ class ParameterDistributionNormal : public VisitableParameterDistribution<Parame
      */
     double get_rand_sample() override
     {
+        //If ub = lb, sampling can only be succesful if mean = lb and dev = 0.
+        //But this degenerate normal distribution is not allowed by the c++ standard.
+        if (m_upper_bound == m_lower_bound) {
```

Comment:
I don't think so. 
- the normal algorithm should work fine as long as there is a tiny difference
- what would the tolerance be, or who would choose it? I think the only generally valid one would be  something like 1e-300, but if anyone is working at that scale, they are already in trouble.

Reply:
I suppose you could still have the fringe case that 

`m_lower_bound > (1+std::numeric_limits<double>::epsilon)*m_upper_bound`

This is the tolerance I would use, unless you say that we already have a safeguard, or don't need one for `m_lower_bound>m_upper_bound`.

Reply:
I think the safeguard for LB > UB is rational expectation. I wouldn't expect that to work, because it's impossible to say what the result should be. But LB = UB is useful and safe, so we handle that case. There could be an `assert(UB >= LB)`, not sure if that exists yet.

Of course it's possible to do some floating point calculation and put the result into the distribution without checking it. But in general, I think floating point rounding errors should be handled where the values are produced if it's reasonable to do so, not where they are used. Otherwise you are throwing around tolerances everywhere. That's why I also changed the computation in `set_params_distributions_normal`.

Reply:
Alright, I am fine with this. I just saw two floating points being compared with `==` and wanted to make sure that this is really what we want.